### DISCUSSION TITLE
Walkthroughs

### DISCUSSION THREAD

**Discussion by worksofliam** [eclipse-theia/theia#11402] [Q&A]
Is it possible to use `contributes.walkthroughs` in a Theia extension? I build an extension for VS Code and wondering if the same Walkthroughs can exist in Theia.

Example: https://github.com/halcyon-tech/vscode-ibmi-walkthroughs/blob/main/package.json#L30

**Comment by vince-fugnitto** [comment]
@worksofliam thank you for the discussion, like any vscode extension you can always try first but I do not believe we support `contributes.walkthroughs` yet.
